pages/index.jsx

```diff
@@ -1,21 +1,141 @@
 import styled from "styled-components";
 import { Routes } from "@config/routes";
+import { color, textFont, displayFont } from "@styles/theme";
+import { Testimonial } from "@features/ui";
+
+const menuItems = [
+  { text: "Home", href: Routes.home },
+  { text: "Products", href: Routes.products },
+  { text: "Documentation", href: Routes.documentation },
+  { text: "Pricing", href: Routes.pricing },
+];
+
+const companyLogosURL = [
+  "company-logos/layers.svg",
+  "company-logos/sisyphus.svg",
+  "company-logos/circooles.svg",
+  "company-logos/catalog.svg",
+  "company-logos/quotient.svg",
+];
+
+const testimonials = [
```

Comment:
This data should come from the API. See this endpoint: https://prolog-api.profy.dev/content-page/home